projects/common/src/time/fixed-time-range.ts

```diff
@@ -28,6 +28,6 @@ export class FixedTimeRange implements TimeRange {
 
```

Comment:
Discussed with @anandtiwary, the values here were a mistake and assigned incorrectly. Fixing them here because part of the PR depends on them. Other than this PR no other places use them